src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -88,6 +88,7 @@ class ScalaSettings extends Settings.SettingGroup {
   val Xshowobj = StringSetting("-Xshow-object", "object", "Show internal representation of object.", "")
   val showPhases = BooleanSetting("-Xshow-phases", "Print a synopsis of compiler phases.")
   val sourceReader = StringSetting("-Xsource-reader", "classname", "Specify a custom method for reading source files.", "")
+  val XnoValueClasses = BooleanSetting("-Xno-value-classes", "Do not use value classes. Helps debugging.")
```

Comment:
Shouldn't it be a -Y option? It's a "compiler hacker" option and it's a bit dangerous since toggling it can break an API since it changes method signatures.